test/conformance/api/v1beta1/util.go

```diff
@@ -340,3 +347,21 @@ func validateImageDigest(imageName string, imageDigest string) (bool, error) {
 
 	return ref.Context().String() == digest.Context().String(), nil
 }
+
+// WaitForScaleToZero will wait for the specified deployment to scale to 0 replicas.
+// Will wait up to 6 times the scaleToZeroGracePeriod (30 seconds) before failing.
+func WaitForScaleToZero(t pkgTest.TLegacy, deploymentName string, clients *test.Clients) error {
```

Comment:
Now given the function is package limited, should it be `waitForScaleToZero`.
Same for alpha.